### PR TITLE
for heavens sake, EMBED THAT

### PR DESCRIPTION
**Pull request by capslock321**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -70,7 +70,9 @@ async def uptime(ctx):
             hours, remainder = divmod(int(delta_uptime.total_seconds()), 3600)
             minutes, seconds = divmod(remainder, 60)
             days, hours = divmod(hours, 24)
-            await ctx.send(f"{days}d, {hours}h, {minutes}m")
+            embed = discord.Embed(title = "Uptime:", color = discord.Colour.red())
+            embed.add_field(f"{days}d, {hours}h, {minutes}m")
```
Comment (Seniatical) [inline bot.py]: Fix that smh
name and value are missing in the field
:p

**Comment by capslock321** [comment]
hold on lemme edit

**Comment by Seniatical** [comment]
ye use a description not a field

**Comment by capslock321** [comment]
changed, take a look nvm lol ur quick

**Comment by Seniatical** [comment]
ok looking good

**Comment by capslock321** [comment]
merggged

**Pull request merged**